### PR TITLE
Support Symfony 5

### LINKED ISSUE #12: Symfony 5 support
The issue to track and coordinate efforts.

### PR DESCRIPTION
**Pull request by GSadee**
Fixes #12

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,22 +22,22 @@
     "require": {
         "php": "^7.2",
 
-        "symfony/config": "^3.4|^4.3",
-        "symfony/dependency-injection": "^3.4|^4.3",
-        "symfony/framework-bundle": "^3.4|^4.3",
-        "symfony/http-kernel": "^3.4|^4.3",
+        "symfony/config": "^4.3",
```
Comment (lchrusciel) [inline composer.json]: Shouldn't we allow for 5.0?
```suggestion
        "symfony/config": "^4.3|^5.0",
```

Also, do we want to remove support for 4.3?

File: src/Bundle/spec/Renderer/Adapter/EmailTwigAdapterSpec.php [hunk 2]
```diff
@@ -20,7 +20,7 @@
 use Sylius\Component\Mailer\Renderer\Adapter\AbstractAdapter;
 use Sylius\Component\Mailer\Renderer\RenderedEmail;
 use Sylius\Component\Mailer\SyliusMailerEvents;
-use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
```
Comment (lchrusciel) [inline src/Bundle/spec/Renderer/Adapter/EmailTwigAdapterSpec.php]: Shouldn't we create a polifil for this behavior?

Comment (lchrusciel) [reply src/Bundle/spec/Renderer/Adapter/EmailTwigAdapterSpec.php]: /cc @pamil

Comment (GSadee) [reply src/Bundle/spec/Renderer/Adapter/EmailTwigAdapterSpec.php]: Added for Event class

**Comment by lchrusciel** [comment]
Thank you, Grzegorz! :1st_place_medal:

**Pull request merged**